tools/manage_clvm.py

```diff
@@ -37,21 +37,60 @@
 top_levels = {"chia"}
 
 
+class ManageClvmError(Exception):
```

Comment:
I wonder why you didn't move this down to the others or the others up to here :)